src/main/webapp/student/scripts/play-game.js

```diff
@@ -147,16 +162,34 @@ function initSubmitButton(gameInstanceId, gameId) {
 }
 
 // Update the question that has changed
-async function updateCurrentQuestion(gameId, questionId) {
-  currentQuestionId = questionId;
+async function updateCurrentQuestion(gameId, questionId, isCurrentQuestionActive) {
+  
+  if(currentQuestionId != questionId) {
+    resetDOM();
+    currentQuestionId = questionId;
+  }
+  currentQuestionActive = isCurrentQuestionActive
 
   let currentQuestionDocRef = db.collection('games').doc(gameId).collection('questions').doc(questionId);
-
   const currentQuestion = await queryCurrentQuestion(currentQuestionDocRef);
 
   // Add the question title to the UI
   createQuestionObject(currentQuestion.title);
 
+  if(!currentQuestionActive){
+      if(!submited){
+        document.getElementById('submitButton').disabled = false;
+        document.getElementById("submitButton").click();
+        document.getElementById('submitButton').disabled = true;
+        document.getElementById("quiz").innerHTML='';
```

Comment:
What is this for? An empty answer is being sent right?

Reply:
Yeah so this means the current question is not active and there wasn't a submission or selection so it is submitted automatically, for example, in case that the time is up and the teacher ends the question, if the student didn't submit an answer it submits null which returns a wrong answer.